### DISCUSSION TITLE
Run Two Campaigns ?

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1808] [Q&A] [answered]
Hi, if I backup the liberation_nextturn file of say Campaign A, can I start a new Campaign B and then return to continue Campaign A later by using the backup liberation_nextturn file ? 

Thanks

**Comment by Starfire13** [comment, marked as answer]
Yes. Though why not just generate a new .miz from your Liberation save when you wanna play that campaign?

**Reply by GrEaSeLiTeNiN** [reply]
What about "autosave.liberation" and "tmpsave.liberation" in the directory Liberation/saves? 
a) What is the difference between these two files? 
b) Are they the current Campaign settings and progress (while "liberation_nextturn.miz" is just the next mission file)?
c) If (b) is true, then should I not backup Campaign A & B in Liberation/saves with unique names like "autosave_CampA.liberation" and "autosave_CampB.liberation" so that when I want to go back to either, I load them first then generate the liberation_nextturn miz file? 

I am thinking of it as being similar to BMS, where you save different campaigns (in the same theater) using unique names, as the autosave gets written over. 

Thanks!

**Reply by Starfire13** [reply]
They are both variants of autosave, just saving at different points. TBH I never use them. I have always saved my campaign files under unique names.

**Reply by GrEaSeLiTeNiN** [reply]
Ok, yeah, so I saved two campaigns using unique names and Lib saves are as follows:
![image](https://user-images.githubusercontent.com/94265170/143679196-868a387f-26af-47d8-84eb-9beacc6364c0.png)
